recipes/jest-to-node-test-runner/src/workflow.test.ts

```diff
@@ -0,0 +1,20 @@
+import { resolve } from "node:path";
```

Comment:
I don't think there are enough tests here, we should make sure the behavior works in cases of
- Jest-like globals from a different library, I.e. playwright
- Other Jest features

Reply:
It's a draft 😉